frontend/src/app/modules/availability/new-slot/schedule-list-item/schedule-list-item.component.ts

```diff
@@ -17,6 +18,11 @@ export class ScheduleListItemComponent implements OnInit {
 
     public endValue: string;
 
+    public scheduleForm = new FormGroup({
+        startTime: new FormControl('16', [Validators.pattern('[0-9]{2}')]),
+        endTime: new FormControl('00', [Validators.pattern('[0-9]{2}')]),
```

Comment:
Validation does not match pattern dd:dd

Reply:
@apofis96 
![image](https://user-images.githubusercontent.com/79774156/186145838-b45d2f32-fee1-4940-97e6-c24b739c96c9.png)
its valid? I can't do its like one string so I do it for each of this two because all logic on this web page was wrotten and hour and minutes input its like separate things